src/lib/features/access/createAccessService.ts

```diff
@@ -17,6 +17,11 @@ import {
     createFakeEventsService,
 } from '../events/createEventsService';
 
+// We need this function to satisfy the type expectations of withTransactional
+export const curriedCreateAccessService =
```

Comment:
withTransactional expects a function that takes config and returns an instantiated service